raiden/tests/integration/network/transport/test_matrix_transport_assumptions.py

```diff
@@ -1,29 +1,26 @@
 from urllib.parse import urlsplit
 
 import pytest
-from eth_utils import encode_hex, to_normalized_address
+from eth_utils import encode_hex, to_checksum_address
 from matrix_client.errors import MatrixRequestError
 
 from raiden.network.transport.matrix.client import GMatrixClient, User
 from raiden.network.transport.matrix.utils import make_client
 from raiden.tests.utils import factories
 from raiden.utils.signer import LocalSigner
+from raiden.network.transport.matrix.utils import make_room_alias, join_broadcast_room, login
 
 # https://matrix.org/docs/spec/appendices#user-identifiers
 USERID_VALID_CHARS = "0123456789abcdefghijklmnopqrstuvwxyz-.=_/"
 
 
 def create_logged_in_client(server: str) -> GMatrixClient:
```

Comment:
You will need to also change the return typing to `Tuple[GMatrixClient, Signer]` or else tests will fail